infrastructure/cdn-in-a-box/traffic_ops/setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -x
+set -e
+
+# First, we need to authenticate
+curl -skc cookie.jar -d "{\"u\":\"$TO_ADMIN_USER\",\"p\":\"twelve\"}" https://localhost:6443/api/1.3/user/login
+echo
```

Comment:
it's not `localhost:6443` -- it's `https://$TO_HOST:$TO_PORT`

Reply:
But surely since this is running _on_ the traffic_ops machine `https://localhost:$TO_PORT` would be fine?